raheem/src/components/ThankYou.js

```diff
@@ -29,42 +34,52 @@ function ThankYou() {
     }, []);
 
     return (
-        <ThankYouContainer>
-            {cancelled &&
-                <Cancelled>
-                    <h2>Thank You</h2>
-                    <p>Thank you for your time today.</p>
-                    <p>If you accidentally clicked on 'Exit', you can click the back button
-                    to return to the report submission form and continue filling out your report.
-                    Otherwise, we hope you have a great rest of the day.
-                    {/* goBack Button */}
-                    </p>
-
-                    <ButtonContainer className="landingButtonContainer">
-                        <BackContainer>
-                            <GoBack />
-                            {/* go to thank you */}
-                        </BackContainer>
-                    </ButtonContainer>
-                </Cancelled>}
-
-            {submitted &&
+        <Container>
+            <Content>
+                <Officer profile={{
+                    officer: "Officer Peyton",
+                    precinct: "#15",
+                    badge: "R4567"
+                }} />
+
+                <hr />
+
+                {cancelled &&
+                    <Cancelled>
+                        <SubHeading>Let us remind you</SubHeading>
+                        <Paragraph>Type in your email and we can email you a reminder, so you can complete your in-depth review later</Paragraph>
+
+                        <p>Email</p>
+                        <input placeholder="email@emailaddress.com"/>
+
+                        <ButtonContainer className="landingButtonContainer">
+                            <BackContainer>
+                                <GoBack />
+                                {/* go to thank you */}
+                            </BackContainer>
+                        </ButtonContainer>
+                    </Cancelled>}
```

Comment:
We can remove this bit because it is now in `Email.js` connected to the backend to save the user's email in a collection.

This component should just be the Thank You portion like you have below based on J and Michelle's Figma designs because if the user cancels their report, they'll find themselves on that `Email.js` component.